### PR TITLE
5794 - Added Busy Indicator Hide Event

### PR DESCRIPTION
**Pull request by tjamesallen15**
**Explain the _details_ for making this change. What existing problem does the pull request solve?**

This PR will add a hide event in Busy Indicator Component.

**Related github/jira issue (required)**:
Closes https://github.com/infor-design/enterprise/issues/5794

**Steps necessary to review your pull request (required)**:
- Pull this branch, build, and run the app
- Go to http://localhost:4000/components/busyindicator/example-stop-event.html
- Open `Developer Console`
- Click `Start` Button
- Click `Stop` Button
- See `Console`

**Included in this Pull Request**:
- [ ] An e2e or functional test for the bug or feature.
- [x] A note to the change log.

### CODE REVIEW

**Comment by tmcconechy** [comment]
works for me - can you test if this will work? @edgarinfor

File: app/views/components/busyindicator/example-stop-event.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+      $('#standalone-busy').trigger('start.busyindicator');
+    });
+
+    $('#busy-complete-trigger').click(function(e) {
+      e.preventDefault();
+      $('#standalone-busy').trigger('complete.busyindicator');
+    });
+
+    $('#standalone-busy').on('hide.busyindicator', () => {
+      console.log('stop event occured');
```
Comment (tmcconechy) [inline app/views/components/busyindicator/example-stop-event.html:36]:
```suggestion
      console.log('hide event fired');
```

**Comment by tjamesallen15** [comment]
> Member

I tried it but the customer wants the trigger to be fired when all of the busy indicator is gone in the DOM. When I tried putting it inside the complete event, it fires okay but the busy indicator is still there in the DOM for a while.

**Review by tmcconechy — APPROVED** [review]
Ok i think this should solve it then.. :)

**Review by deep7102 — APPROVED** [review]

**Pull request merged**